driver/razerchromacommon.c

```diff
@@ -1213,8 +1213,8 @@ struct razer_report razer_chroma_misc_set_dpi_xy(unsigned char variable_storage,
     struct razer_report report = get_razer_report(0x04, 0x05, 0x07);
 
     // Keep the DPI within bounds
-    dpi_x = clamp_u16(dpi_x, 100, 30000);
-    dpi_y = clamp_u16(dpi_y, 100, 30000);
+    dpi_x = clamp_u16(dpi_x, 100, 35000);
+    dpi_y = clamp_u16(dpi_y, 100, 35000);
```

Comment:
Let me know if this should be parameterized or a new function instead.

Reply:
no that's fine, we regularly bump this limit because Razer keeps thinking higher number = better